timetracker/entries/models.py

```diff
@@ -17,8 +17,9 @@ class Project(models.Model):
     def __str__(self):
         return '<{}> {}'.format(self.client, self.name)
 
+    def get_entries(self):
```

Comment:
Whilst this is a very clean encapsulation, Django already provides reverse accessors like this for all relation fields.

See https://docs.djangoproject.com/en/1.8/topics/db/queries/#following-relationships-backward